src/banking/primitive/core/ServerSolution.java

```diff
@@ -10,13 +10,13 @@
 
 class ServerSolution implements AccountServer {
 
-	static String fileName = "accounts.ser";
-
+	static final String fileName = "accounts.ser";

```

Comment:
Hey Jake, looks like "filename" should be "FILENAME" instead. Strange how you left just one error...everything else is right. I sense a strong dark force pulling the error strings.